streaming/Sink.ml

```diff
@@ -246,7 +250,7 @@ type ('top, 'a, 'b) flat_map =
   } -> ('top, 'a, 'b) flat_map
 
 
-let _flat_map f (Sink top) =
+let flat_map f (Sink top) =
```

Comment:
This is already corrected in `master` (https://github.com/odis-labs/streaming/commit/8c746ba15cadb4ea863a821499377b0526d4bd68), but thanks for noticing!